a5py/ascot5io/wall.py

```diff
@@ -437,15 +452,15 @@ def move_component(self, movement, direction, component):
         movement    How much the component is moved in metres
         direction    Which direction the component is moved [x, y, z]
         component     Bool array of the points of a component
-        wall         Wall object where the wall is copied from, if left empty active 
+        wall         Wall object where the wall is copied from, if left empty active
         wall from filename will be used
 
         @return the new wall data
         """
         rwall = self.read()
 
         #Movement in metres
-        t = movement/np.sqrt(direction[0]**2 + direction[1]**2 + direction[2]**2) 
+        t = movement/np.sqrt(direction[0]**2 + direction[1]**2 + direction[2]**2)
```

Comment:
There is no argument `wall` in the function. Are you sure these changes are part of the commit and not accidental?

Reply:
No changes were made there. Basically, my editor automatically deletes trailing spaces, and I could not get them to revert (with the trailing spaces). I could only revert one of them, but if I then tried to revert the other, the first trailing space automatically got reverted. :(